data/strings/types_strings.txt

```diff
@@ -3768,7 +3671,7 @@
     uk = Взуття
 
   [type.recycling.green_waste]
-    en = Green/Organic Waste
+    en = Organic Waste
```

Comment:
Please leave green here, some people may not know what is Organic in this context.

Reply:
Also this doesn't really make sense.   Presumably it's not waste that is certified organic and might be food or might be grass clipppings.    I'm guessing it is grass, brush and leaves.

Reply:
just any type of organic waste. i think this would be understood but better to keep 'green' just in case

Reply:
I am boggled by "any type of organic waste".   To me that means anything that is a plant or animal product and could include grass/leaves/brush, decaying food, and human waste (e.g. from the blackwater tank in an RV/caravan).   I have never encountered a place with blanket acceptance and I wouldn't call it green.

Reading the wiki, I get it now.  There is `garden_waste` which is the leaves/etc. I am thinking of, and `food_waste` for food.  That leaves out bodily fluids.    So `green_waste` is something that will not make sense to people in the US.   There is "yard waste" which is grass/leaves/etc..  And there is "food waste composting" which is what the `food_waste` tag is about, and an example pickup service is https://blackearthcompost.com/.

If I had to describe a place that took both, I would say "yard and food waste composting".

In the US I generally do not see "recycling" applied to composting.  Recyling has a connotation of chemical engineering to reduce to raw materials which are then used in manufacturing.